src/s2a_moms_tent.tw2

```diff
@@ -0,0 +1,287 @@
+* Patch ripped wall in L.R.
+* Talk to Ron about budget (<u>no more</u> wasting money at The Gutter!!)
+And sure enough, there was a line item for me. It read:
+* <i>Talk to $playerName about the house rules — NO MORE SNEAKING OFF WITH ANNA AFTER DARK!</i>
+Great. The one time I disappear without a trace, she’s somehow [[sober enough to notice|Mom's Tent - table hub]].
+
+::Mom's Tent - support beam
+The support beam separating the kitchen from my tent was looking pretty funky, but it was still standing. I scanned from the bottom up, watching the series of notches and dates go by:
+
+|RON 69/666
+|
+|
+|
+|
+|
+|$playerName 7/20XX
+|
+|
+|
+|$playerName 13/20XX
+|
+|
+|$playerName 9/20XX
+|
+|$playerName 3/20XX
+|
+|
+|$playerName 2/20XX
+
+From looking around the scene, it’s the only proof I ever lived here.
+
+I felt that ball of rage in my gut flare up again. Fuck this place. And fuck the assholes who live here. I’m so beyond done with this shit.
+
+I pounded (if: $beerDrank is true)[the rest of the watery piss that passes for beer by Ron’s standards](else:)[the sugary dregs of the Redpop], crunched the can, and left it on the table. They could clean up their own messes here. I was done with that shit.
+
+Right. Ok. I just had to [[get my shit together|Mom's Tent - your room]] and get outta here.
+
+::Mom's Tent - your room
+The tarp separating my bed from the kitchen fluttered gently in the draft. Something beyond it caught my eye.
+
+A bare foot. Lying on my bed.
+
+Oh shit. Mom’s still here. I froze and waited what felt like an eternity and listened.
+
+(click: "waited what felt like an eternity and listened")[(transition: "dissolve") +(transition-time: 8s)[Snoring]]
+
+(click: "Snoring")[She’s out, thank god. My head was still murky as hell and a thrumming headache was gaining momentum; the last thing I needed was to be chewed out for something that wasn’t even my fault. Jesus, Anna; why’d you do this to me, anyway?
+
+Focusing again, I remembered why I was here. My eyes darted around the room, panic rising in my chest, until I saw it: [[my backpack|Mom's Tent - journal]].]
+
+::Mom's Tent - journal
+I quickly rummaged through my shit to make sure it was how I left it. My journal was there, but the cash was gone. I must’ve had it on me last night when I got drugged. Did someone take it?
+
+Whatever. I grabbed the pack and quietly slipped it over my shoulder.
+
+(click: "slipped it over my shoulder")[ <center>[(transition: "pulse")[<span style="display: block; white-space: nowrap;">//[[**Journal obtained**|Mom's Tent - poster choice]]//</span>]]</center>]
+
+::Mom's Tent - poster choice
+I looked up from where I’d found the bag and my eyes came to rest on a poster hanging over my bed. It read:
+
+>>(link: "DOUBLE TROUBLE: Wicked Wrestlin’ Federation Invitational")[(set: $wrestling to true)(go-to: "Mom's Tent - looking at mom")]
+>>(link: "Open Mic Battle Royale - TONIGHT ONLY")[(set: $rapping to true)(go-to: "Mom's Tent - looking at mom")]
```

Comment:
Minor detail, but: since these variables describe the player, I'd argue we should rename them to `$isWrestler`/`$isRapper` or just `$wrestler`/`$rapper`.

Reply:
Great point! I wasn't sure about naming conventions for variables, but $isWrestler / $isRapper make sense to me.